ide/static/js/login.js

```diff
@@ -50,7 +50,8 @@ class Login extends React.Component {
       contentType: false,
       data: {
         username: username,
-        password: password 
+        password: password,
+        fabrik_login_system: true
```

Comment:
Rename this to `isOAuth`

Reply:
If this method is called from `componentDidMount()` then pass `true` value otherwise `false`. It will me changed when we add remember me feature

Reply:
@Jatana Can you make this change, then I can test it end-to-end

Reply:
@Ram81 Why it will be changed?

Reply:
Let's say you logged in using github and then closed Fabirk, now you re-open Fabrik after sometime and we try to verify the user is logged in or not and it works. But if I logout and try using username and password then also we go into that block of code for oauth. Does that make sense?